addons/website/static/src/js/utils.js

```diff
@@ -416,6 +416,24 @@ function generateGMapLink(dataset) {
         + '&ie=UTF8&iwloc=&output=embed';
 }
 
+/**
+ * Checks if the current device is mobile.
```

Comment:
Comment could be better: "Checks if the edited content is currently previewed as in a mobile device"?